package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Proxy",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "node bundle"
```

Comment:
you can save an FS operation by specifying the path:
```node bundle.js```
however this will suffice